hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/HStore.java

```diff
@@ -276,7 +276,8 @@ protected HStore(final HRegion region, final ColumnFamilyDescriptor family,
     // used by ScanQueryMatcher
     long timeToPurgeDeletes =
         Math.max(conf.getLong("hbase.hstore.time.to.purge.deletes", 0), 0);
-    LOG.trace("Time to purge deletes set to {}ms in store {}", timeToPurgeDeletes, this);
+    LOG.trace("Time to purge deletes set to {}ms in store {} in region {}", timeToPurgeDeletes,
```

Comment:
In all the places we have this pattern  "in store {} in region {}"
Better we can change toString().  Include region and store name there.  Then in every log we can just refer to 'this'
<Region Name>:<Store Name>  some format like this for toString?

Reply:
I like what @anoopsjohn  says. toString could be encodedRegionName/store-name (region name is too much -- could be anything given it includes start key... could be a spew of binary characters....)

Reply:
Otherwise I'd comment that no need of words 'store' or 'region' in log message; no need of these qualifiers.

Reply:
Good suggestions, I will do more work on this, thank you for all your reviews